rdbms/src/main/java/org/spine3/server/storage/jdbc/command/query/InsertCommandQuery.java

```diff
@@ -20,41 +20,45 @@
 
 package org.spine3.server.storage.jdbc.command.query;
 
-import org.spine3.server.command.storage.CommandStorageRecord;
+import org.spine3.server.command.CommandRecord;
 
+import static org.spine3.server.storage.jdbc.Sql.BuildingBlock.BRACKET_CLOSE;
+import static org.spine3.server.storage.jdbc.Sql.BuildingBlock.BRACKET_OPEN;
+import static org.spine3.server.storage.jdbc.Sql.BuildingBlock.COMMA;
+import static org.spine3.server.storage.jdbc.Sql.BuildingBlock.SEMICOLON;
+import static org.spine3.server.storage.jdbc.Sql.Query.INSERT_INTO;
+import static org.spine3.server.storage.jdbc.Sql.Query.VALUES;
+import static org.spine3.server.storage.jdbc.Sql.nPlaceholders;
 import static org.spine3.server.storage.jdbc.command.query.CommandTable.COMMAND_COL;
 import static org.spine3.server.storage.jdbc.command.query.CommandTable.COMMAND_STATUS_COL;
 import static org.spine3.server.storage.jdbc.command.query.CommandTable.ID_COL;
 import static org.spine3.server.storage.jdbc.command.query.CommandTable.TABLE_NAME;
 
-
 /**
- * Query that inserts a new {@link CommandStorageRecord} to the {@link CommandTable}.
+ * Query that inserts a new {@link CommandRecord} to the {@link CommandTable}.
  *
  * @author Alexander Litus
  * @author Andrey Lavrov
  */
 public class InsertCommandQuery extends WriteCommandRecordQuery {
 
-    @SuppressWarnings("DuplicateStringLiteralInspection")
     private static final String QUERY_TEMPLATE =
-            "INSERT INTO " + TABLE_NAME + " (" +
-                    ID_COL + ", " +
-                    COMMAND_STATUS_COL + ", " +
-                    COMMAND_COL +
-                    ") VALUES (?, ?, ?);";
+            INSERT_INTO + TABLE_NAME + BRACKET_OPEN +
+            ID_COL + COMMA +
+            COMMAND_STATUS_COL + COMMA +
+            COMMAND_COL + BRACKET_CLOSE +
+            VALUES + nPlaceholders(3) + SEMICOLON;
 
     private InsertCommandQuery(Builder builder) {
         super(builder);
     }
 
-
     public static Builder newBuilder() {
         final Builder builder = new Builder();
         builder.setStatusIndexInQuery(2)
-                .setIdIndexInQuery(1)
-                .setRecordIndexInQuery(3)
-                .setQuery(QUERY_TEMPLATE);
+               .setIdIndexInQuery(1)
```

Comment:
In my last review I have asked to find some peace for these indexes. The suggestion was to have an enum with self-explanatory member names.

Please process, or let's discuss.